### PR TITLE
Add TMDb guidelines

### PR DESCRIPTION
**Pull request by almazhankapan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -74,4 +74,21 @@ You can now use ``client_id`` and ``client_secret`` for making Trakt API calls.
 
 For example, if your client id is 123456, the file would read:
 
-> API_KEY_TRAKT=123456
\ No newline at end of file
+> API_KEY_TRAKT=123456
+
+### Tmdb API Setup
```
Comment (sam-eng) [inline CONTRIBUTING.md:79]: Looks great! The only nitpicky thing is I think The Movie Database capitalizes their abbreviation as `TMDb`--if you could change this, that'd be great!

Comment (almazhankapan) [reply CONTRIBUTING.md:79]: yes, sounds good, will edit that

**Review by sam-eng — APPROVED** [review]
Thanks Alma, looks good!

**Comment by almazhankapan** [comment]
> Thanks Alma, looks good!

thanks!

**Pull request merged**